contracts/implementations/tableland/DeviceDefinitionTable.sol

```diff
@@ -128,30 +132,42 @@ contract DeviceDefinitionTable is AccessControlInternal {
      * @dev The caller must be the owner of the manufacturer ID
      * @dev The specified Device Definition Table must exist
      * @dev The pair (model,year) must be unique
-     * @param tableId The unique identifier of the Device Definition Table to be associated
+     * @param manufacturerId The unique identifier of the manufacturer
      * @param data Input data with the following fields:
      *  id -> The alphanumeric ID of the Device Definition
      *  model -> The model of the Device Definition
      *  year -> The year of the Device Definition
      *  metadata -> The metadata stringfied object of the Device Definition
      */
     function insertDeviceDefinition(
-        uint256 tableId,
+        uint256 manufacturerId,
         DeviceDefinitionInput calldata data
     ) external {
         ITablelandTables tablelandTables = TablelandDeployments.get();
-        string memory prefix = DeviceDefinitionTableStorage
-            .getStorage()
-            .prefixes[tableId];
+        DeviceDefinitionTableStorage.Storage
+            storage dds = DeviceDefinitionTableStorage.getStorage();
+        uint256 tableId = dds.tables[manufacturerId];
```

Comment:
Could we bail here if this is zero? Can Tableland tables be burned?

Reply:
The contracts does not allow tables burning. We would catch that in the `ownerOf` below anyway.